Mighty-Menorahs/reactive-nutshell
# Ep edit

# Description

Please check if you can click on the title of the task and it goes to an edit task page. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ X] New feature (non-breaking change which adds functionality)

# Testing Instructions

git fetch --all
git checkout ep-edit
Please check if you can click on the title of the task and it goes to an edit task page. 




# Checklist:
